src/components/createDocumentForm/addDocumentButton/addDocumentButton.tsx

```diff
@@ -0,0 +1,30 @@
+import Box from '@mui/material/Box'
+import Button from '@mui/material/Button'
+
+interface AddDocumentButtonProps {
+  onAddDocument: () => void
+  disabled: boolean
+}
+
+export const AddDocumentButton = (props: AddDocumentButtonProps) => {
```

Comment:
можно пропсы спредить сразу в аргументах
```suggestion
export const AddDocumentButton = ({onAddDocument, disabled}: AddDocumentButtonProps) => {
```

Reply:
Поскольку в проекте нет соглашений по этому поводу, я предпочитаю не использовать деструктуризацию в параметрах компонента